src/bootloaders/grub2.c

```diff
@@ -312,6 +340,7 @@ static bool grub2_write_config(const BootManager *manager, const Kernel *default
                 .os_id = os_id,
                 .is_separate = is_separate,
                 .submenu = false,
+                .manager = manager,
```

Comment:
This is the assignment. Where it is used?

Reply:
line 268 in the same file